app/views/feedbacks/new.html.erb

```diff
@@ -28,6 +28,12 @@
       <%= ts("We value anonymous feedback, but if you'd like a personal reply to your question or comment, we need your email address to contact you. Our spam filter does collect IP addresses, but we never see them.") %>
     </p>
     <dl>
+      <dt><%= f.label :username, ts("Your name (optional)") %></dt>
+      <dd>
+        <%= f.text_field :username, size: 60 %>
```

Comment:
You can take out the size attribute here and on the email field -- we handle the input size with CSS now. Also, there's no need to close the dl tag -- it should be
```
    <dl>
      <dt><%= f.label :username, ts("Your name (optional)") %></dt>
      <dd>
        <%= f.text_field :username %>
      </dd>
      <dt><%= f.label :email, ts("Your email (optional)") %></dt>
      <dd>
        <%= f.text_field :email, "aria-describedby" => "email-field-description" %>
      </dd>
    </dl>
```